src/SignalR/clients/csharp/Client.Core/src/HubConnectionBuilder.cs

```diff
@@ -25,7 +27,7 @@ public HubConnectionBuilder()
             Services = new ServiceCollection();
             Services.AddSingleton<HubConnection>();
             Services.AddLogging();
-            this.AddNewtonsoftJsonProtocol();
+            this.AddJsonProtocol();
```

Comment:
I don't think we're going to be removing Newtonsoft as the default in preview4 in the end. Let's swap this back and make sure we show how to replace Newtonsoft with System.Text.Json in a blog post.

Reply:
I doubt we'll be getting this in for preview4

Reply:
> I don't think we're going to be removing Newtonsoft as the default in preview4 in the end. 

> I doubt we'll be getting this in for preview4

Why?

Reply:
> I don't think we're going to be removing Newtonsoft as the default in preview4 in the end.

Discussed with @ahsonkhan offline. There are still things missing that will make the experience pretty rough (https://github.com/aspnet/AspNetCore/issues/8489).

Reply:
> There are still things missing that will make the experience pretty rough

Particularly around dictionary and collection interfaces like IReadonlyList.

Fyi, @steveharter, @layomi, @joshfree